packages/vector_graphics/lib/vector_graphics.dart

```diff
@@ -1,7 +1,96 @@
-library vector_graphics;
+import 'dart:typed_data';
+import 'dart:ui' as ui;
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+import 'package:vector_graphics_codec/vector_graphics_codec.dart';
 
-/// A Calculator.
-class Calculator {
-  /// Returns [value] plus 1.
-  int addOne(int value) => value + 1;
+import 'src/listener.dart';
+
+const VectorGraphicsCodec _codec = VectorGraphicsCodec();
+
+/// A widget that displays a vector_graphics formatted asset.
+class VectorGraphicsWidget extends StatefulWidget {
+  const VectorGraphicsWidget.asset({ Key? key, required this.assetName }) : super(key: key);
```

Comment:
Let's not do this.

I copied this model from `Image`, and it is a mistake. We should just take the `Uint8List` asset here. We should delegate retrieving those bytes to other parts of the program that can decide when/where/how/what isolate to do this on.